client/account/info.ctrl.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function (io, $mdToast, errorHandler) {
```

Comment:
Could you change this to use classes/prototype functions now that the other controllers also do that?